overrides/main.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block announce %}
+{% if config.extra.version_warning %}
+<div class="versionwarning" style="margin: .6rem auto; padding: 0 .8rem">
+  <h1 style="color: #ffcc00">You are viewing documentation for Knative version: {{ config.extra.knative_version }}</h1>
+  <p>
+  Knative {{ config.extra.knative_version }} documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see <a href="http://knative.dev/docs">the latest version</a>.
```

Comment:
Is better to make this relative `href="http://knative.dev/docs"`
This way it works from github pages, or locally

Reply:
it's a little tricky to do it relatively since the include is static but could be added to any pages. I might need to think about this one a bit (it's probably an easy solution Im just not thinking of).

Reply:
I think using `href="../docs` should work and cover HTTPS

Reply:
This works for me
![image](https://user-images.githubusercontent.com/1094878/118499520-0447f700-b6f5-11eb-9288-ffcf31e36a84.png)

But the HTML generated should preserve it to `../docs` and not `knative.dev/docs`

Reply:
hmm weird using `../docs` generates `<a href="http://knative.dev/docs">`
![image](https://user-images.githubusercontent.com/1094878/118500240-a23bc180-b6f5-11eb-925c-f6f6684de6aa.png)

How can we make it stick to `../docs`

Reply:
I guess make it `../docs` and we can fix later maybe with some javascript

Reply:
I dont think ../docs will work because you might be on /docs/getting-started/foo/bar/baz in which case that doesnt go to the right place

Reply:
ok, it's a bit of a hack, but pushed something that I think will work